packages/common/src/entities/wallet.ts

```diff
@@ -12,6 +13,9 @@ export class Wallet {
     @Column()
     name: string;
 
+    @Column({ type: "enum", enum: CurrencyTypeEnum, default: CurrencyTypeEnum.Eth })
```

Comment:
Default here doesn't really make sense, one should just always be passwed

```suggestion
    @Column({ type: "enum", enum: CurrencyTypeEnum })
```